cloud.foundry.cli/src/main/java/cloud/foundry/cli/Main.java

```diff
@@ -1,21 +1,12 @@
 package cloud.foundry.cli;
```

Comment:
I would remove the Main.class and set the main-method into the BaseController. 
So you would reduce the complexity by one class, which just have a singleMethod, which is responsible to call the BaseController.

Reply:
I think that's a good idea, the `BaseController` is pretty empty, too.

Reply:
Done.